src/FsSpreadsheet.Exceljs/Cell.fs

```diff
@@ -0,0 +1,88 @@
+﻿namespace FsSpreadsheet.Exceljs
+
+
+module JsCell =
+
+    open Fable.Core
+    open Fable.Core.JsInterop
+    open FsSpreadsheet
+    open Fable.ExcelJs
+
+    [<Emit("console.log($0)")>]
+    let private log (obj:obj) = jsNative
+
+    let writeFromFsCell (fsCell: FsCell) = 
+        match fsCell.DataType with
+        | Boolean   -> 
+            fsCell.ValueAsBool() |> box |> Some
+        | Number    -> 
+            fsCell.ValueAsFloat() |> box |> Some
+        | Date      -> 
+            /// Here it will actually show the correct DateTime. But when writing, exceljs will apply local offset. 
+            let dt = fsCell.ValueAsDateTime() |> System.DateTimeOffset 
+            /// Therefore we add offset and it should work.
+            let dt = dt + dt.Offset |> box |> Some
+            dt
+        | String    -> 
+            fsCell.Value |> Some
+        | anyElse ->
+            let msg = sprintf "ValueType '%A' is not fully implemented in FsSpreadsheet and is handled as string input." anyElse
+            #if FABLE_COMPILER_JAVASCRIPT
+            log msg
+            #else
+            printfn "%s" msg
+            #endif
+            fsCell.Value |> box |> Some 
+
+    /// <summary>
+    /// `worksheetName`, `rowIndex` and `columnIndex` are only used for debugging.
+    /// </summary>
+    /// <param name="worksheetName"></param>
+    /// <param name="rowIndex"></param>
+    /// <param name="columnIndex"></param>
+    /// <param name="jsCell"></param>
+    let readToFsCell worksheetName rowIndex columnIndex (jsCell: Cell) =
```

Comment:
Naming (I think this is better as the one in ExcelIO). 

Maybe just let as is for now and clean up when the actual need arises.